docs/side_quests/metadata.md

```diff
@@ -0,0 +1,1039 @@
+[[id:sampleB, character:tux], /workspaces/training/side-quests/metadata/data/guten_tag.txt]
+[[id:sampleC, character:sheep], /workspaces/training/side-quests/metadata/data/hallo.txt]
+[[id:sampleD, character:turkey], /workspaces/training/side-quests/metadata/data/hello.txt]
+[[id:sampleE, character:stegosaurus], /workspaces/training/side-quests/metadata/data/hola.txt]
+[[id:sampleF, character:moose], /workspaces/training/side-quests/metadata/data/salut.txt]
+[[id:sampleG, character:turtle], /workspaces/training/side-quests/metadata/data/ciao.txt]
+[[id:sampleB, character:tux], de]
+[[id:sampleA, character:squirrel], fr]
+[[id:sampleC, character:sheep], de]
+[[id:sampleD, character:turkey], en]
+[[id:sampleE, character:stegosaurus], es]
+[[id:sampleF, character:moose], fr]
+[[id:sampleG, character:turtle], it]
+```
+
+We can see that the meta map is the first element in each map and the map is the same for both channels. We can simply use the `join` operator to combine the two channels:
+
+=== "After"
+
+    ```groovy title="main.nf" linenums="20" hl_lines="11-12"
+    workflow  {
+
+      ch_samplesheet = Channel.fromPath("./data/samplesheet.csv")
+                      .splitCsv(header: true)
+                      .map { row ->
+                          [ [id:row.id, character:row.character], row.recording ]
+                      }
+
+      ch_prediction = IDENTIFY_LANGUAGE(ch_samplesheet)
+
+      ch_languages = ch_samplesheet.join(ch_prediction)
+                                   .view()
+    }
+    ```
+
+=== "Before"
+
+    ```groovy title="main.nf" linenums="20"
+    workflow  {
+
+        ch_samplesheet = Channel.fromPath("./data/samplesheet.csv")
+                    .splitCsv(header: true)
+                    .map { row ->
+                        [ [id:row.id, character:row.character], row.recording ]
+                    }
+                    .view()
+
+        ch_prediction = IDENTIFY_LANGUAGE(ch_samplesheet)
+        ch_prediction.view()
+
+    }
+    ```
+
+```bash title="View joined channel"
+nextflow run main.nf
+```
+
+```console title="View joined channel"
+[[id:sampleA, character:squirrel], /workspaces/training/side-quests/metadata/data/bonjour.txt, fr]
+[[id:sampleB, character:tux], /workspaces/training/side-quests/metadata/data/guten_tag.txt, de]
+[[id:sampleC, character:sheep], /workspaces/training/side-quests/metadata/data/hallo.txt, de]
+[[id:sampleD, character:turkey], /workspaces/training/side-quests/metadata/data/hello.txt, en]
+[[id:sampleE, character:stegosaurus], /workspaces/training/side-quests/metadata/data/hola.txt, es]
+[[id:sampleF, character:moose], /workspaces/training/side-quests/metadata/data/salut.txt, fr]
+[[id:sampleG, character:turtle], /workspaces/training/side-quests/metadata/data/ciao.txt, it]
+```
+
+It is becoming a bit hard to see, but if you look all the way on the right side, you can see that now each of our language predictions is associated with our input files.
```

Comment:
```suggestion
It is becoming a bit hard to see, but if you look all the way on the right side, you can see that now each of our language predictions is associated with our input files.
Each channel is not composed of 3 items:

1. The meta map (`[id:sampleA, character:squirrel]`)
2. The input file (`/workspaces/training/side-quests/metadata/data/bonjour.txt`)
3. The language identifier (`fr`)
```

Just clarify, rather than highlighting the difficulty in parsing.